packages/vats/src/core/lib-boot.js

```diff
@@ -141,7 +142,9 @@ export const makeBootstrap = (
       }
       rawBootstrap(vats, devices).catch(e => {
         console.error('BOOTSTRAP FAILED:', e);
-        throw e;
+        // Terminate because the vat is irrecoverable state.
+        // The error can't be throw because the outer 'bootstrap' function already returned.
```

Comment:
It probably doesn't cost much to throw after calling exit (I don't believe it's like `process.exit()` where no other instruction executes after).

Also typo

```suggestion
        // Terminate because the vat is in an irrecoverable state.
        // The error can't be thrown because the outer 'bootstrap' function already returned.
```